src/Storages/IStorage.h

```diff
@@ -444,21 +444,15 @@ class IStorage : public std::enable_shared_from_this<IStorage>, public TypePromo
       return false;
     }
 
-    struct ImportStats
-    {
-        ExecutionStatus status;
-        std::size_t elapsed_ns = 0;
-        std::size_t bytes_on_disk = 0;
-        std::size_t read_rows = 0;
-        std::size_t read_bytes = 0;
-        std::string file_path = "";
-    };
-
+    /*
+      It is kind of hard to describe this API. It is currently only implemented in StorageObjectStorage.
```

Comment:
```suggestion
It is currently only implemented in StorageObjectStorage.
```

It's very hard to comment on that comment :) 

But try to make the comments helpful for the the future readers of the code. Don't make them think that 'even author of the code can't explain what it is about' :)